components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -50,4 +61,34 @@ describe('LeagueEntries', () => {
     );
     expect(leagueEntryNumber).toHaveTextContent('Entry 3');
   });
+
+  it('renders teamLogo when user makes a pick', () => {
+    const teamLogoUrl = 'https://example.com/logo.png';
+
+    render(
+      <LeagueEntries
+        entryName="Entry 2"
+        linkUrl=""
+        isPickSet={true}
+        teamLogo={teamLogoUrl}
+      />,
+    );
+
+    const leagueEntryContainerCard = screen.getByTestId(
+      'league-entry-container-card',
+    );
+    const leagueEntryNumber = screen.getByTestId('league-entry-number');
+    const entryStatus = screen.getByTestId('entry-status');
+    const leagueEntryPickButton = screen.getByTestId(
+      'league-entry-pick-button',
+    );
+    const leagueEntryLogo = screen.getByTestId('league-entry-logo');
+
+    expect(entryStatus).toHaveTextContent('alive');
+    expect(leagueEntryContainerCard).toBeInTheDocument();
```

Comment:
I don't you need to test if the container card is in the document if you'd already tested for children elements of it.

Reply:
Good catch. Removed the container card.